integration/docker/README.md

```diff
@@ -1,18 +1,38 @@
-## Building docker image
+## Prerequisite
+To build your Alluxio Docker image, a Docker 19.03+ is required. 
+
+## Building docker image for production
 To build the Alluxio Docker image from the default remote url, run
 
 ```console
-$ docker build -t alluxio .
+$ docker build -t alluxio/alluxio .
 ```
 
 To build with a local Alluxio tarball, specify the `ALLUXIO_TARBALL` build argument
 
 ```console
-$ docker build -t alluxio --build-arg ALLUXIO_TARBALL=alluxio-${version}.tar.gz .
+$ docker build -t alluxio/alluxio --build-arg ALLUXIO_TARBALL=alluxio-${version}.tar.gz .
+```
+
+Starting from v2.6.0, alluxio-fuse image is deprecated. It is embedded in alluxio/alluxio image.
+
+## Building docker image for development
+Starting from now, Alluxio has a separate image for development, which pre-installs more tools. 
```

Comment:
more tools for what purposes?
can you give more info here?
like the default alluxio docker image optimize to only install packages needed for Alluxio services to run.
This dev installs some development tools like.....